README.md

```diff
@@ -67,25 +56,25 @@ The directory structure of your new project looks like this:
 │   ├── __init__.py    <- Makes src a Python module
 │   │
 │   ├── data           <- Scripts to download or generate data
-│   │   └── make_dataset.py
 │   │
 │   ├── features       <- Scripts to turn raw data into features for modeling
-│   │   └── build_features.py
 │   │
 │   ├── models         <- Scripts to train models and then use trained models to make
-│   │   │                 predictions
-│   │   ├── predict_model.py
-│   │   └── train_model.py
-│   │
+│   │                     predictions
 │   └── visualization  <- Scripts to create exploratory and results oriented visualizations
-│       └── visualize.py
 │
+├── create_notebook_config.py <- script for creating Jupyter notebook configuration.
 └── tox.ini            <- tox file with settings for running tox; see tox.testrun.org
 ```
 
-## Contributing
+### Useful Make rules
```

Comment:
Approve with one change (you don't need to get another review before merging):
It's probably worth mentioning that simply typing `make` will list all available commands (but it excludes those rules associated with running the analysis; applies only to technical commands like this).